security/list-users.php

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Login</title>
+</head>
+<body>
+<a href="index.php"> Take me home, country roads ... </a> <br>
+<?php
+session_start();
+if (!isset($_SESSION["email"])) {
+    die("Only authenticated users are allowed");
+}
+
+include "db-connection.php";
+$conn = openCon();
+
+$sql = "SELECT * from person where email = '" . $_SESSION["email"] . "';";
+$resultSet = $conn->query($sql) or die("Failed to query from DB!");
```

Comment:
I like that you have added die clause, but it would be better if you prepare the sql query with prepare() function and after that execute it with execute() function. You can leave the die clause but add it to execute() function.